scripts/pull-images-from-ecr-on-ec2.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# AWS parameters
+AWS_REGION="us-east-2"  # Change to your region
+ACCOUNT_ID="975050215890"  # Replace with your AWS account ID
+REPO_NAME="rei-engine-ecr-repo"
+
+# List of images to build and push
+IMAGES=(
+  "zimba-rei-common-components_deals-api"
+#  "zimba-rei-common-components_investor-profiles-api"
+#  "zimba-rei-common-components_listings-api"
+#  "zimba-rei-common-components_mortgages-api"
+#  "zimba-rei-common-components_nginx"
+#  "zimba-rei-common-components_subscriptions-api"
+#  "zimba-rei-common-components_underwritings-api"
```

Comment:
should these be commented out?

Reply:
Not finally. These scripts are here just for convenience and are not yet used in CI/CD. I will update them when we get to automating the process.